src/coreclr/vm/codeman.h

```diff
@@ -247,12 +259,92 @@ typedef struct _hpCodeHdr
     }
 #endif // FEATURE_EH_FUNCLETS
 
+
+#ifdef DACCESS_COMPILE
+    void EnumMemoryRegions(CLRDataEnumMemoryFlags flags, IJitManager* pJitMan);
+#endif  // DACCESS_COMPILE
+
+};
+
+typedef DPTR(RealCodeHeader) PTR_RealCodeHeader;
+typedef DPTR(InterpreterRealCodeHeader) PTR_InterpreterRealCodeHeader;
+
+#ifdef FEATURE_INTERPRETER
+
+struct InterpreterCodeHeader
+{
+    PTR_InterpreterRealCodeHeader   pRealCodeHeader;
+
+public:
+    PTR_BYTE GetDebugInfo()
+    {
+        SUPPORTS_DAC;
+        return pRealCodeHeader->phdrDebugInfo;
+    }
+    PTR_EE_ILEXCEPTION GetEHInfo()
+    {
+        return pRealCodeHeader->phdrJitEHInfo;
+    }
+    PTR_BYTE GetGCInfo()
+    {
+        SUPPORTS_DAC;
+        return pRealCodeHeader->phdrJitGCInfo;
+
+    }
+    PTR_MethodDesc GetMethodDesc()
+    {
+        SUPPORTS_DAC;
+        return pRealCodeHeader->phdrMDesc;
+    }
+#if defined(FEATURE_GDBJIT)
+    VOID* GetCalledMethods()
+    {
+        SUPPORTS_DAC;
+        return pRealCodeHeader->pCalledMethods;
```

Comment:
pCalledMethods isn't defined in interpreter real code header is it? copy-paste error?

Reply:
I used to have that on the code header even for the interpreter, but removed it later based on a feedback. I'll delete this leftover.